cura/Settings/MachineManager.py

```diff
@@ -132,6 +117,19 @@ def __init__(self, parent = None):
         if containers:
             containers[0].nameChanged.connect(self._onMaterialNameChanged)
 
+        # NEW
```

Comment:
Remove `# NEW`. In a few years it won't be new any more.

Reply:
Removed in 632be1132